test-e2e/bastion_tunnel_test.go

```diff
@@ -68,7 +68,7 @@ func TestIzeTunnelUp(t *testing.T) {
 
 	ize := NewBinary(t, izeBinary, examplesRootDir)
 
-	stdout, stderr, err := ize.RunRaw("tunnel", "up")
+	stdout, stderr, err := ize.RunRaw("tunnel", "up", "--ssh-public-key", "~/.ssh/id_rsa_tunnel_test.pub")
```

Comment:
Doesn't seem right. It should contain both private and public key in order for the test to be correct